### PR TITLE
Template override digests

### PR DESCRIPTION
**Pull request by ronalfy**
This allows a third-party plugin to provide an absolute URL to a template file and override the template digest emails.

### CODE REVIEW

File: core/email-batches/comment-email-batch.php [hunk 1]
```diff
@@ -166,6 +174,14 @@ public function __construct( $comment, Prompt_Comment_Flood_Controller $flood_co
 		if ( ! apply_filters( 'prompt/send_comment_notifications', true, $this->prompt_comment, $recipient_ids ) )
 			return null;
 
+		/**
+		 * Filter recipient IDs to be sent comment notifications.
+		 *
+		 * @param array $recipient_ids Array of recipient IDs.
+		 * @param object $comment
+		 */
+		$recipient_ids = apply_filters( 'prompt/send_comment_notifications/recipient_ids', $recipient_ids, $this->prompt_comment );
```
Comment (cyberhobo) [inline core/email-batches/comment-email-batch.php]: What is this filter for? It doesn't seem related to template overrides and there is already a similar prompt/comment/recipient_ids filter.

Comment (ronalfy) [reply core/email-batches/comment-email-batch.php]: @cyberhobo filter removed. It wasn't being used anyway.

https://github.com/postmatic/postmatic-basic/pull/134/commits/b5130b424a9d23e2c8f811a795f50423a00b9b87
